src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/StorePal.cs

```diff
@@ -264,7 +264,15 @@ private static void LoadMachineStores()
 
                     ICertificatePal pal;
 
-                    while (OpenSslX509CertificateReader.TryReadX509Pem(fileBio, out pal) ||
+                    // Some distros ship with two variants of the same certificate.
+                    // One is the regular format ('BEGIN CERTIFICATE') and the other
+                    // contains additional AUX-data ('BEGIN TRUSTED CERTIFICATE').
+                    // The additional data contains the appropriate usage (e.g. emailProtection, serverAuth, ...).
+                    // Because corefx doesn't validate for a specific usage, derived certificates are rejected.
+                    // For now, we skip the certificates with AUX data and use the regular certificates.
+                    // In the future, we should look at validating with a specific usage and prefer
+                    // the trusted certificates over the regular certificates when loading the store.
```

Comment:
Please remove these last two sentences.  It would require a completely different architecture to our chains to enable that; and as-is it sounds like a free-form TODO.